Makefile

```diff
@@ -114,6 +114,10 @@ hosts-chromium:
 	            ln -sfv "$(LIB_DIR)/browserpass/hosts/chromium/$(APP_ID).json" "/etc/chromium/native-messaging-hosts/$(APP_ID).json"; \
 	            [ -e "/etc/chromium/native-messaging-hosts/$(APP_ID).json" ] || echo "Error: the symlink points to a non-existent location" >&2; \
 	            ;; \
+	FreeBSD)    mkdir -p "${PREFIX}/etc/chrome/native-messaging-hosts/"; \
```

Comment:
Do you know if the same applies to OpenBSD? I'm just wondering if we can replace `FreeBSD)` with `*BSD)`